main.js

```diff
@@ -0,0 +1,22 @@
+window.addEventListener('DOMContentLoaded', () => {
+    let elem = document.getElementsByClassName('books')[0];
+    Books.map(book => {
+
+        elem.innerHTML += `
```

Comment:
It is dangerous and generally bad practice to modify innerHTML. We should do this with handlebars anyway